Packages/vcs/Lib/VTKPlots.py

```diff
@@ -1485,11 +1508,18 @@ def png(self, file, width=None,height=None,units=None,draw_white_background = Tr
         writer.SetInputConnection(imgfiltr.GetOutputPort())
         writer.SetFileName(file)
         writer.Write()
+        if manager:
+          self.renWin.AddRenderer(manager.renderer)
 
   def cgm(self,file):
         if self.renWin is None:
           raise Exception,"Nothing to dump aborting"
 
+        from vtk_ui.manager import get_manager
```

Comment:
Yeah, I can add a function to extract the UI and one to add it back in. Would save some repeated logic.